src/cache/precomputed_quantities.jl

```diff
@@ -288,7 +413,16 @@ function set_precomputed_quantities!(Y, p, t)
         # @. ᶜK += Y.c.sgs⁰.ρatke / Y.c.ρ
         # TODO: We should think more about these increments before we use them.
     end
-    @. ᶜts = ts_gs(thermo_args..., ᶜspecific, ᶜK, ᶜΦ, Y.c.ρ)
+    @. ᶜts =
+        t > 0 ?
+        ts_gs(
+            thermo_args...,
+            ᶜspecific,
+            ᶜK,
+            ᶜΦ,
+            Y.c.ρ,
+            nothing,
+        ) : ts_gs(thermo_args..., ᶜspecific, ᶜK, ᶜΦ, Y.c.ρ)
```

Comment:
Maybe it's this? I would change this to:
```julia
    if t > 0
    @. ᶜts =
        ts_gs(
            thermo_args...,
            ᶜspecific,
            ᶜK,
            ᶜΦ,
            Y.c.ρ,
            nothing,
        )
else
    @. ᶜts = ts_gs(thermo_args..., ᶜspecific, ᶜK, ᶜΦ, Y.c.ρ)
end
```
so that the runtime if-else block is not inside the broadcast expression.